theano/tensor/tests/test_opt.py

```diff
@@ -3555,9 +3555,9 @@ def test2(self):
         # we have to change the following we should not see tensor.Alloc
         # in op_classes and we have to change the assert.
         assert tensor.Alloc in op_classes
+        # self.assertTrue(check_stack_trace(f, ops_to_check=[tensor.Alloc]))
 
         # Check stacktrace was copied over correctly after opt was applied
```

Comment:
Please update the comment, and put it before the commented-out "self.assertTrue(...)".